core/google/api/core/gapic_v1/method.py

```diff
@@ -82,9 +116,9 @@ class _GapicCallable(object):
             False, should accept a timeout argument. If metadata is not
             False, it should accept a metadata argument.
         retry (google.api.core.retry.Retry): The default retry for the
-            callable. If False, this callable will not retry.
+            callable. If falsy, this callable will not retry.
```

Comment:
This is less true than before, it's an explicit `is False` check, not a `if not retry:` check

Reply:
Should be fixed